addons/website/static/src/js/editor/mega_menu.js

```diff
@@ -0,0 +1,95 @@
+odoo.define('website.editor.megamenu', function (require) {
+'use strict';
+
+const core = require('web.core');
+const publicWidget = require('web.public.widget');
+
+publicWidget.registry.hoverableDropdown.include({
```

Comment:
This widget is already a website widget... so don't `include`: modify it where needed directly :+1: